CONTRIBUTING.md

```diff
@@ -103,7 +96,7 @@ Examples:
     Consensus: Add new opcode for BIP-XXXX OP_CHECKAWESOMESIG
     Net: Automatically create hidden service, listen on Tor
     Qt: Add feed bump button
-    Trivial: Fix typo in init.cpp
+    Logs: Fix typo in log message
 
 Note that translations should not be submitted as pull requests, please see
 [Translation Process](https://github.com/bitcoin/bitcoin/blob/master/doc/translation_process.md) 
```

Comment:
nit: extra whitespace at the end of this line that can be removed while touching this file.

Reply:
nit: also 3 extra whitespaces at the end of all the 4 last lines of this file except the last one, that can be removed while touching this file